schemas/v1.0/azure.yaml.json

```diff
@@ -525,6 +525,15 @@
                     "$ref": "#/definitions/workflow"
                 }
             }
+        },
+        "cloud": {
+            "type": "object",
+            "title": "The cloud configuration used for the project.",
+            "description": "Optional. Provides additional configuration for deploying to sovereign clouds such as Azure Government.",
```

Comment:
Default is AzureCloud? Shouldn't we also say that?

Reply:
Description updated.